tests/configs/modules/alert/default.js

```diff
@@ -3,7 +3,13 @@
  * By rejas
  * MIT Licensed.
  */
-let config = require(process.cwd() + "/tests/configs/default.js").configFactory({
+let config = {
+	port: 8080,
```

Comment:
Do these config infos need to be in each file? Cant they be refactored into a common file?

Reply:
tell me how, `require` cannot be used in the browser ...

Reply:
Solution is simple, we can remove all the default values which are defined in `js/default.js`. Will add a commit.

Reply:
Yeah, I thought about using import but then a lot more would have to be refactored IIRC